samples/sqltransport-sqlpersistence/SqlServer_3/Sender/App.config

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <appSettings>
+    <add key="NServiceBus/Outbox"
+         value="true" />
+  </appSettings>
+  <!--startcode SenderConnectionStrings-->
+  <connectionStrings>
+    <add name="NServiceBus/Transport"
```

Comment:
given we explicitly dont want them to use different connections for transport and persister. i think merge these into a single entry